nix/klayout.nix

```diff
@@ -71,38 +73,44 @@ in clangStdenv.mkDerivation {
     qtxmlpatterns
     curl
     gcc
+    libgit2
   ];
 
   propagatedBuildInputs = [
     ruby
   ];
-  
-  buildPhase = ''
-    mkdir -p $out/lib
-    echo "Using $NIX_BUILD_CORES threads…"
+
+  configurePhase = ''
+    if [ "${if stdenv.isDarwin then "1" else "0" }" = "1" ]; then
+      export LDFLAGS="-headerpad_max_install_names"
+    fi
     ./build.sh\
-      -option -j$NIX_BUILD_CORES\
+      -prefix $out/lib\
       -without-qtbinding\
       -python $(which python3)\
       -ruby $(which ruby)\
-      -prefix $out/lib\
       -expert\
-      -verbose
-    runHook postBuild
+      -verbose\
+      -dry-run
   '';
-
-  postBuild = if stdenv.isDarwin then ''
-    mkdir $out/bin
-    cp $out/lib/klayout.app/Contents/MacOS/klayout $out/bin/
-  '' else ''
-    mkdir $out/bin
-    cp $out/lib/klayout $out/bin/
+  
+  buildPhase = ''
+    echo "Using $NIX_BUILD_CORES threads…"
+    make -j$NIX_BUILD_CORES -C build-release PREFIX=$out
```

Comment:
ohh. so `build.sh` is used to configure and not actually build?

Reply:
wait I just noticed `--dry-run`. so you are actually not relying on `build.sh` for building ?

Reply:
Yep, it's just a configure now